src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/RegisterDSFAppSI.cs

```diff
@@ -34,7 +37,8 @@ public class RegisterDSFAppSI : IDSF
             ILogger<RegisterDSFAppSI> logger,
             IHttpContextAccessor httpContextAccessor,
             IOptionsMonitor<AppSettings> settings,
-            HttpClient httpClient)
+            HttpClient httpClient,
+            IAccessTokenGenerator accessTokenGenerator, IAppResources appResource)
```

Comment:
Ideally we would have introduced these dependencies on a different level. Maybe a new class as a typed HttpClient. Not sure if it is important enough to trigger even more work on this issue. 
It is more the principle to avoid introducing dependencies that are unnecessary.

Reply:
We just changed it to be like this because this is the recommended way